cmd/rig-operator/main.go

```diff
@@ -98,86 +102,97 @@ func run(cmd *cobra.Command, _ []string) error {
 
 	restConfig := ctrl.GetConfigOrDie()
```

Comment:
Can we include the rest of the function until here in FX as well for more consistency?

Reply:
Yep, done :)